apps/meteor/client/views/omnichannel/contactInfo/EditContactInfo.tsx

```diff
@@ -188,14 +188,14 @@ const EditContactInfo = ({ contactData, onClose, onCancel }: ContactNewEditProps
 
 	if (isLoadingCustomFields) {
 		return (
-			<ContextualbarContent>
-				<FormSkeleton />
-			</ContextualbarContent>
+			<ContextualbarDialog onClose={onClose}>
+				<ContextualbarSkeleton />
+			</ContextualbarDialog>
```

Comment:
I noticed you didn't wrap the others `ContextualbarSkeleton` with `ContextualbarDialog`. Did you mean to wrap this one?

Reply:
this will return two `Contextualbar` one from `ContextualbarDialog` another from `ContextualbarSkeleton`